src/markdown-pages/add-ons/antrea.md

```diff
@@ -5,8 +5,17 @@ linktitle: "Antrea"
 weight: 30
 title: "Antrea Add-On"
 addOn: "antrea"
+isDeprecated: true
 ---
 
+## Deprecation Notice
+
+### This add-on is deprecated.
+
+As of March 27, 2023, kURL no longer intends to continue to offer this add-on as part of the ongoing kURL project. This add-on is now considered deprecated, and may no longer be offered by the project after June 31st, 2023. Existing installs that use this add-on will be best effort supported during this deprecation window. Please consider using the [Flannel](https://kurl.sh/docs/add-ons/flannel) add-on for your CNI needs moving forward.
```

Comment:
```suggestion
Antrea add-on is deprecated, and may no longer be offered by the project after June 31st, 2023. Existing installs that use this add-on will be best effort supported during this deprecation window. Please consider using the [Flannel](https://kurl.sh/docs/add-ons/flannel) add-on for your CNI needs moving forward.
```
What about we try to make it less verbose?
Do we need the first part?

Reply:
I prefer having the starting date for the deprecation. We can point to this and say to existing vendors that they had ~90 day notice to transition and make changes to their spec.